pkg/cmd/config/get/get.go

```diff
@@ -25,13 +24,10 @@ func NewCmdConfigGet(f *cmdutil.Factory, runF func(*GetOptions) error) *cobra.Co
 	}
 
 	cmd := &cobra.Command{
-		Use:   "get <key>",
-		Short: "Print the value of a given configuration key",
-		Example: heredoc.Doc(`
-			$ gh config get git_protocol
-			https
-		`),
-		Args: cobra.ExactArgs(1),
+		Use:     "get <key>",
+		Short:   "Print the value of a given configuration key",
+		Example: "$ gh config get git_protocol",
+		Args:    cobra.ExactArgs(1),
```

Comment:
Aside from the long example, there is a weird undiscussed observation from maintainers where some extension authors and people are setting things that are not actually official core `gh` configurations.  If we decide to touch this, then having a `Long` here or somewhere that explains what people should expect with configuration; maybe this live in the root `config` command 🤔